dasharo-security/secure-boot.robot

```diff
@@ -253,4 +355,28 @@ Prepare Test Files
         ...    ${BAD_FORMAT_NAME}
         ...    ${BAD_FORMAT_URL}
         ...    ${BAD_FORMAT_SHA256}
+        Download To Host Cache
```

Comment:
@tym2k1 I do not think that `Prepare Test Files` keyword is used anywhere, those changes are not needed. Now, we are getting files using `Download ISO And Mount As USB`

Reply:
Addressed and squashed